Commands/gemini3.py

```
@@ -148,20 +116,30 @@ def reply_with_grounded_gemini(self, message):
-            m = prefix + m
-        if i == len(result) - 1 and duck_urls:
-            m += f" 📝 Source(s): {' | '.join(duck_urls)}"
-        self.send_privmsg(channel, m)
-        time.sleep(1)
+    try:
+        clean_result = clean_str(result, ['`', '*'])
+        send_chunks(self.send_privmsg, channel, clean_result)
+        self.send_privmsg(channel, f"📝 Source(s): {' | '.join(duck_urls)}")
+    except Exception as e:
+        print(f"[Error] {e}")
+        self.send_privmsg(f"Failed to send a response. Please try again later")
```

Ruff (0.11.9)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)